integration_tests/run.sh

```diff
@@ -113,7 +114,10 @@ for function_name in "${all_functions[@]}"; do
     fi
 done
 
+now=$(date +"%r")
 echo "Sleeping $LOGS_WAIT_SECONDS seconds to wait for logs to appear in CloudWatch..."
+echo "This should be done in 10 minutes from $now"
```

Comment:
This message is a nice touch, I think it improves the user experience of running the tests.